lib/Optimizer/Transforms/DecompositionPatterns.cpp

```diff
@@ -269,6 +269,32 @@ struct SToPhasedRx : public OpRewritePattern<quake::SOp> {
   }
 };
 
+// quake.s target
+// ────────────────────────────────────
+// Gate equivalence:
```

Comment:
nit: The intention of the line in these comments originally was to serve as an "equals" (like in elementary school).

```console
     2
   +1
   ──
    3
```

Reply:
Thanks for the info about the semantics of these comments :) Fixed them accordingly.